presto-docs/src/main/sphinx/sql/alter-function.rst

```diff
@@ -0,0 +1,49 @@
+==============
+ALTER FUNCTION
+==============
+
+Synopsis
+--------
+
+.. code-block:: none
+
+    ALTER FUNCTION qualified_function_name [ ( parameter_type[, ...] ) ]
+    RETURNS NULL ON NULL INPUT | CALLED ON NULL INPUT
+
+
+Description
+-----------
+
+Alter the definition of an existing function.
+
+Parameter type list must be specified if multiple signatures exist
+for the specified function name. Otherwise, the query would fail.
+
+If exactly one signature exists for the specified function name,
+parameter type list might be omitted.
+
+A subset of routine characteristics, as defined in
```

Comment:
Maybe say that right now only modifying the null-call clause is supported.  Otherwise it's pretty surprising as a user to get to the end of this documentation and discover that mostly you can't modify the function.